common/religions/shamanism_group.txt

```diff
@@ -173,54 +188,57 @@ shamanism_group = {
 
 		icon = 75
 		heresy_icon = 76
-		
+
 		color = { 0 220 183 }
-		
+
 		scripture_name = THE_STORIES
 		priest_title = TIDESAGE
-		
+
 		high_god_name = GOD_TIDEMOTHER
-		
+
 		god_names = {
 			GOD_TIDEMOTHER
 		}
-		
+
 		evil_god_names = {
 			THE_CHAOS
 		}
-		
+
 		# Intermarries
 		intermarry = old_ways
 		intermarry = light_group
 		intermarry = primitive_shamanism
 		intermarry = secular_group
 		intermarry = shamanism_group
-		
+
 		# Modifiers
 		character_modifier = {
 			orcish_fel_opinion = -10
 			burning_legion_religion_opinion = -200
 			death_god_opinion = -200
 			old_gods_worship_opinion  = -200
-			
+
 			# Must be in every religion
 			uncolonized_trait_opinion = -200
 		}
-		
-		# War laws
-		can_call_crusade = no
-		
+
 		# Secular laws
+		can_excommunicate = yes
```

Comment:
\>Head of Religion title is secular ruler
\>Can Excommunicate
\> Instead of having a reason to imprison character, you can pay with piety and then freely execute character
\>+15 (up to infinity) opinion from vassal for every excommunication -> lift excommunication

Reply:
True, it's OP. Removed.